crates/meilisearch/tests/batches/mod.rs

```diff
@@ -167,17 +168,18 @@ async fn list_batches_status_filtered() {
 async fn list_batches_type_filtered() {
     let server = Server::new().await;
     let index = server.index("test");
-    let (task, _) = index.create(None).await;
-    index.wait_task(task.uid()).await.succeeded();
-    let (task, _) = index.delete().await;
+    let (task,_status_code) = index.create(None).await;
     index.wait_task(task.uid()).await.succeeded();
+    index
+        .add_documents(serde_json::from_str(include_str!("../assets/test_set.json")).unwrap(), None)
+        .await;
```

Comment:
Hey, this test is wrong now.
I think it comes down to a bad rebase, it's one of the test that I recently updated because it was flaky (and it still is after your PR :sweat_smile:).
Can you put back my version here